pkg/uuid/mock.go

```diff
@@ -28,6 +29,11 @@ func (up *uuidProviderMock) ID() (string, error) {
 	return fmt.Sprintf("%s%012d", Prefix, up.counter), nil
 }
 
+func (up *uuidProviderMock) FromString(input string) (string, error) {
+
+	return "", errors.New("Error uuidProviderMock does not implement FromString")
```

Comment:
Do we need this error? Maybe simply `nil` here

Reply:
I also thought of removing it. But interesting when I changed it into
return "", nil
the go test was failing.  Let me try more and will update you on that.  For now, I left it as it is.